### PR TITLE
[AC-449] Uncaught java.lang.NullPointerException in LoginFragment

### PR DESCRIPTION
**Pull request by Arkadeep-sophoIITG**
Resolved uncaught exception in showToast method in LoginFragment. 
https://issues.openmrs.org/browse/AC-449

### CODE REVIEW

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java [hunk 1]
```diff
@@ -425,12 +423,14 @@ public void setLocationErrorOccurred(boolean errorOccurred) {
 
     @Override
     public void showToast(String message, ToastUtil.ToastType toastType) {
-        ToastUtil.showShortToast(getContext(), toastType, message);
+        if (getActivity() != null)
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java]: Could you put braces `{` `}` for the `if` statement as advised at https://wiki.openmrs.org/display/docs/Java+Conventions?

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java [hunk 2]
```diff
@@ -425,12 +423,14 @@ public void setLocationErrorOccurred(boolean errorOccurred) {
...
+            ToastUtil.showShortToast(getActivity(), toastType, message);
     }
 
     @Override
     public void showToast(int textId, ToastUtil.ToastType toastType) {
-        ToastUtil.showShortToast(getContext(), toastType, getResources().getString(textId));
+        if (getActivity() != null)
```
Comment (f4ww4z) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/login/LoginFragment.java]: Could you put braces `{` `}` on the `if` statement as advised at https://wiki.openmrs.org/display/docs/Java+Conventions?

**Comment by Arkadeep-sophoIITG** [comment]
@f4ww4z  I made the changes as requested !! Please review it again.

**Comment by DefCon-007** [comment]
@Arkadeep-sophoIITG  Do mention the JIRA issue link in the description. Have a look at https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Pull request merged**